src/compressor/brotli/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+# brotli
+
+set(brotli_sources
+	CompressionPluginBrotli.cc
+	BrotliCompressor.cc
+)
+include(ExternalProject)
+ExternalProject_Add(brotli_ext
+	DOWNLOAD_DIR ${CMAKE_BINARY_DIR}/src/
+	GIT_REPOSITORY "https://github.com/google/brotli.git"
+	GIT_TAG "master"
+	SOURCE_DIR ${CMAKE_BINARY_DIR}/src/brotli
+	CONFIGURE_COMMAND ./configure-cmake --disable-debug 
+	INSTALL_COMMAND ""
+	BUILD_COMMAND $(MAKE)
+	BUILD_IN_SOURCE 1
+	INSTALL_COMMAND "")
+
+ExternalProject_Add_Step(brotli_ext forcebuild
+	DEPENDEES configure
+	DEPENDERS build
+    COMMAND "true"
+	ALWAYS 1)
+
+add_library(brotlicommon STATIC IMPORTED)
```

Comment:
nit, could name it `brotli::common` and use a loop to set the dependency and the properties of these libraries.